src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -247,9 +247,21 @@ <h3>Headers</h3>
                     <!-- /ko -->
                     <!-- ko if: !header.options || header.options.length === 0 -->
                     <div class="input-group">
+                        <!-- ko if: !header.secret -->
                         <input type="text" autocomplete="off" class="form-control form-control-sm" placeholder="value"
                             spellcheck="false" aria-label="Header value"
                             data-bind="event: { keyup: $component.updateRequestSummary }, textInput: header.value">
+                        <!-- /ko -->
+                        <!-- ko if: header.secret -->
+                        <input autocomplete="off" class="form-control form-control-sm input-password"
+                            placeholder="value" spellcheck="false" aria-label="Header value"
+                            data-bind="attr: {type: header.inputTypeValue}, textInput: header.value">
+                        <button data-bind="click: () => $component.toggleSecretHeader(header)"
+                            class="toggle-secrets-button">
```

Comment:
`class="toggle-secrets-button"` - this is still specific case naming (please see my previous comments). It could be just `.btn`:

```
.input-group {
    .btn {
        ...
    }
}
```
An [example from Bootstrap](https://getbootstrap.com/docs/5.1/forms/input-group/#button-addons):
```
<div class="input-group mb-3">
  <button class="btn btn-outline-secondary" type="button" id="button-addon1">Button</button>
  <input type="text" class="form-control" placeholder="" aria-label="Example text with button addon" aria-describedby="button-addon1">
</div>
```